swift-readme.md

```diff
@@ -0,0 +1,30 @@
+##Swift Object Store
```

Comment:
Maybe put this in the openshift readme?

https://github.com/FNNDSC/pfioh/blob/master/openshift/README.rst